.github/workflows/ci-for-default-ns.yaml

```
@@ -75,15 +75,15 @@ jobs:
           # for testing start-debug and stop-debug deployment
           # mons
           kubectl rook_ceph debug start rook-ceph-mon-a
-          sleep 5
+          tests/github-action-helper.sh wait_for_pod_running rook-ceph-mon-a-debug rook-ceph running
           kubectl rook_ceph debug stop rook-ceph-mon-a
           # osd
           kubectl rook_ceph debug start rook-ceph-osd-0
-          sleep 5
+          tests/github-action-helper.sh wait_for_pod_running rook-ceph-osd-0-debug rook-ceph running
           kubectl rook_ceph debug stop rook-ceph-osd-0
 
-          # sleep again so OSD can start
-          sleep 5
+          # check if osd 0 is started
+          tests/github-action-helper.sh wait_for_pod_running rook-ceph-osd-0 rook-ceph deleted
```

Shouldn't we wait for the osd pod to be running again? It's the debug pod that should be deleted.
```suggestion
          tests/github-action-helper.sh wait_for_pod_running rook-ceph-osd-0 rook-ceph running
          tests/github-action-helper.sh wait_for_pod_running rook-ceph-osd-0-debug rook-ceph deleted
```

---

So the flow is like this 
```
if [[ $STATUS == "deleted" ]]
  then
    debug_pod=$(kubectl get pods -n ${NAMESPACE} | grep ${DEPLOYMENT} | awk '!/rook-ceph-osd-0-debug/' | awk '{ print $1  }')
  else
    debug_pod=$(kubectl get pods -n ${NAMESPACE} | grep ${DEPLOYMENT} | awk '{ print $1  }')
```

If it is `deleted` it will skip picking `rook-ceph-osd-0-debug`

---

Actually, I had some changes which would avoid `running` and `deleting` paramter,

It will just be hardcoded for "rook-ceph-osd-0"
```
if [[ $DEPLOYMENT == "rook-ceph-osd-0" ]]
  then
    debug_pod=$(kubectl get pods -n ${NAMESPACE} | grep ${DEPLOYMENT} | awk '!/rook-ceph-osd-0-debug/' | awk '{ print $1  }')
```